### PR TITLE
Adding documentation and additional predicate for sequence localization, see #427

### PR DESCRIPTION
**Pull request by cmungall**

### CODE REVIEW

File: biolink-model.yaml [hunk 1]
```diff
@@ -1374,6 +1374,14 @@ slots:
       - SEMMEDDB:LOCATION_OF
       - WIKIDATA:276
 
+  has sequence location:
+    is_a: related to
+    description: >-
+      holds between two genomic entities when the subject can be localized in sequence coordinates on the object. For example, between an exon and a chromosome.
```
Comment (TomConlin) [inline biolink-model.yaml:1380]: where do less formalized objects such as contigs stand?

Comment (TomConlin) [reply biolink-model.yaml:1380]: Also noting that avoiding overloading this to mean "this is necessarily the actual sequence"  could help.

File: biolink-model.yaml [hunk 2]
```diff
@@ -4668,6 +4691,8 @@ classes:
       object:
         aliases: ['reference']
         range: genomic entity ## typically a chromosomel use monochrom
```
Comment (TomConlin) [inline biolink-model.yaml:4693]: chromosomel

**Comment by deepakunni3** [comment]
@cmungall Added strand and updated description of properties.
Also moved `faldo:location` mapping from `GenomicSequenceLocalization` association to `has_sequence_location` relation.

**Pull request merged**